data/brands/amenity/fast_food.json

```diff
@@ -1475,6 +1475,31 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Exki",
+      "id": "exki-a3472e",
+      "locationSet": {
+        "include": [
+          "be",
+          "es",
+          "fr",
+          "it",
+          "lu",
+          "nl"
+        ]
+      },
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Exki",
+        "brand:wikidata": "Q251760",
+        "brand:wikipedia": "fr:Exki",
+        "cuisine": "healthy;sandwich;salad;pie",
+        "diet:vegan": "yes",
+        "diet:vegetarian": "yes",
```

Comment:
>only
>    only provides this dietary option (N.B. it may often be helpful to tag a vegetarian place with both diet:vegetarian=only and diet:vegan=yes)

https://wiki.openstreetmap.org/wiki/Key:diet

Try to ensure that whether kind of vegan diet this fastfood is ?

Reply:
In fact, I don't know, I have seen this on multiples Exki nodes with overpass turbo :grin:.

Reply:
From a logical perspective, **"Vegan"** is satisfied with the more stringent vegetarian vegetarian (from Wiki's introduction to this, I am not vegetarian, so my understanding is only limited to Osmwiki), so I can In order to meet strict conditions, it is **certainly** satisfied **"vegetarian"**， so use **only**

——————
从逻辑的角度讲，“vegan”满足的是更严格的没有任何动物制品的素食（从维基的介绍上来看是这样的，我并非素食者所以我的了解也仅限于osmwiki的介绍），因此能满足严格的情况下，是一定满足较为宽松的“vegetarian”的

Reply:
But if every plate are not vegetarian then they must not all be vegan.
I have here a counter-example: LIGURIA Linguine, cooked ham without added nitrite, Grana Padano, arugula salad.

Reply:
So if it is not a vegan restaurant， just add the **vegetarian** tag

https://wiki.openstreetmap.org/wiki/Key:diet:vegetarian

![图片](https://user-images.githubusercontent.com/42690037/119864439-35be8f00-bf4d-11eb-85c3-51863dd328b2.png)

You can judge what kind of tag should be carried out according to the standards shown in the figure, combined with your local knowledge

---

BTW，the osm data is strictly differentiated for vegetarian food. There are currently diet types for separate pages, which are only vegan, vegetarian, halal, and kosher

Reply:
According to your more detailed description, the more suitable tag is only a piece of
```
        "diet:vegetarian": "yes",
```

Reply:
> So if it's not a vegan restaurant, just add the vegetarian tag.

:+1: It's not an exclusively vegan or vegetarian restaurant, but they do have some purely vegan (so vegetarian) dishes and some vegetarian (but not vegan). But they also have "normal" dishes.

<details>
<summary>To illustrate here is part of the <a href="https://www.exki.be/sites/default/files/a3_produits_spring_2020_be_frnl_0.pdf">menu (fr/nl)</a> translated here.</summary>

* VEGETABLE PIES Brie-spinach, Zucchini-salmon, Eggplant-tomato-goat... [*no specific label*]
* CASABLANCA Duo of rice and falafels, oriental aubergine compote [*vegan, vegetarian, low lactose*]
* ADANA Risotto of small spelt, bell pepper, tomato and thyme tapenade, Grana Padano [*vegetarian*]

</details>

So for what I understand from the osm wiki the original tagging seems correct ?
```json
        "diet:vegan": "yes",      // some vegan products
        "diet:vegetarian": "yes", // some vegetarian as well
```
If not, I think the tagging as-is is not wrong.

Thanks @LaoshuBaby for the review.